Makefile

```diff
@@ -169,6 +169,14 @@ bin/calico-felix-$(ARCH): $(SRC_FILES) $(LOCAL_BUILD_DEP)
 	     sh -c 'go build -v -i -o $@ -v $(BUILD_FLAGS) $(LDFLAGS) "$(PACKAGE_NAME)/cmd/calico-felix"'; \
 	fi
 
+bin/calico-felix-race-$(ARCH): $(SRC_FILES) $(LOCAL_BUILD_DEP)
+	@echo Building felix with race detector enabled for $(ARCH) on $(BUILDARCH)
+	mkdir -p bin
+	if [ "$(SEMAPHORE)" != "true" -o ! -e $@ ] ; then \
```

Comment:
It's an optimisation to avoid rebuilding the binary on the semaphore FV machines (otherwise restoring the binaries from the cache makes them look stale to Make).  Maybe we should `touch` them instead.

Reply:
Thanks, I understand now.